nrf_modem/doc/rai.rst

```diff
@@ -0,0 +1,162 @@
+.. _release_assistance_indication:
+
+Release Assistance Indication (RAI)
```

Comment:
I don't think this page should be at this level, It should be part of `sockets.rst` page instead.
This level is for high-level library features, like interfaces, tracing etc.

Reply:
I agree with Emanuele's comment. I had this thought earlier, but since the document has a lot of content, we could have a reference from line no:164 of the socket.rst doc to this page?

Reply:
Good point, however the `sockets.rst` file is expanding constantly and it may become increasingly difficult to maintain. What do you think @divipillai about the creation of a `sockets` subdirectory and maintaining `sockets.rst` with references to subdocuments contained in the new folder? Features that could go there could be for example `PDN` and `RAI` for now.

Reply:
Agree with you. This is something followed here also: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/asset_tracker_v2/README.html. We have one more layer of subpages for Application module: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/asset_tracker_v2/doc/modules.html.

Reply:
@divipillai I'm opening a task for moving `RAI` and `PDN` into a `sockets` subdirectory. In the meantime, for this commit, I will merge `rai.rst` into `sockets.rst`.